lesson10/js/main.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const menuContainer = document.getElementById("js-menu_container");
+const menuLists = document.createElement("ul");
+const menuContents = [
+  {
+    to: "bookmark.html",
+    img: "images/1.png",
+    alt: "画像1",
+    text: "ブックマーク",
+  },
+  {
+    to: "message.html",
+    img: "images/2.png",
+    alt: "画像2",
+    text: "メッセージ",
+  },
+];
+
+const loading = document.getElementById("js-loading");
+
+const loadingImg = document.createElement("img");
+loadingImg.src = "images/loading-circle.gif";
+loadingImg.classList.add("el_loadingImg");
```

Comment:
this lines call with function

Reply:
I made loading a function.
I wrote the variable name "loading" outside the function because I wanted to use it to delete the loading image.

[3a7e4eb](https://github.com/nyattsu72/hanson-frontend-cource/pull/13/commits/3a7e4ebd9a983f3a4bf669dacb0231fe6c57ec54)